arrow/locales.py

```diff
@@ -95,14 +106,21 @@ def describe(self, timeframe, delta=0, only_distance=False):
 
         return humanized
 
-    def describe_multi(self, timeframes, only_distance=False):
+    def describe_multi(
+        self,
+        timeframes: List[List[Any]],
+        only_distance: bool = False,  # TODO Check type
+    ) -> str:
         """Describes a delta within multiple timeframes in plain language.
 
         :param timeframes: a list of string, quantity pairs each representing a timeframe and delta.
```

Comment:
If timeframes is a list of string, quantity pairs, we should probably put a concrete type